### DISCUSSION TITLE
When the inventory is full, purchased components are placed in the circuit box but are not displayed there.

### DISCUSSION THREAD

**Discussion by pavel4win** [FakeFishGames/Barotrauma#14931] [Bug reports]
### Disclaimers

- [X] I have searched the issue tracker to check if the issue has already been reported.
- [ ] My issue happened while using mods.

### What happened?

My inventory was full. I had an empty circuit box in my inventory.
When I bought several components, they went into the box. When I attached this box to the wall, they were not there, but their number was shown.

### Reproduction steps

Put the empty circuit box in your inventory.
Fill the rest of the inventory and hands with some items.
go to the seller and buy several electrical components.
They will NOT fall on the floor.
You will see that they went into the box.
Attach the box to the wall
You will not find the purchased components there, although their number is shown.

### Bug prevalence

Happens every time I play

### Single player or multiplayer?

Multiplayer hosted from the in-game menu (= using a listen server)

### Version

v1.6.17.0 (Unto the Breach Update)

### Which operating system did you encounter this bug on?

Windows

**Comment by Rokvach** [comment]
Fixed and tested the fix to be working correctly, fix coming in the next update. Closing discussion.
